react/forms/step3.md

```diff
@@ -0,0 +1,68 @@
+Now we'll modify our form to include a title dropdown.
+
+## Task
+
+1) Make sure the App.js file is still open
+
+2) To create a title dropdown, we'll add a new `value1` attribute to state and a `this.options` array in the constructor.
+
+<pre class="file" data-target="clipboard">
+constructor(props) {
+  super(props);
+  this.state = {
+    value1: 'please choose',
+    value2: '',
+    value3: '',
+    value4: ''
+  };
+  this.onChange = (value1, event) => {
+    this.setState({ value1 });
+  };
+  this.handleTextInputChange1 = value2 => {
+    this.setState({ value2 });
+  };
+  this.handleTextInputChange2 = value3 => {
+    this.setState({ value3 });
+  };
+  this.handleTextInputChange3 = value4 => {
+    this.setState({ value4 });
+  };
+  this.options = [
+    { value: 'please choose', label: 'Please choose', disabled: false },
+    { value: 'miss', label: 'Miss', disabled: false },
+    { value: 'mrs', label: 'Mrs', disabled: false },
+    { value: 'ms', label: 'Ms', disabled: false },
+    { value: 'mr', label: 'Mr', disabled: false },
+    { value: 'dr', label: 'Dr', disabled: false },
+    { value: 'other', label: 'Other', disabled: false }
+  ];
+}
+</pre>
+
+3) Next, we'll add `value1` to the state declaration in the `render()` method:
+
+<pre class="file" data-target="clipboard">
+render() {

```

Comment:
I don't believe we want to copy `render() {` to the clipboard. Instead show / ask the user to locate `render() {` and then copy the variables.